product/interfaces/graphql/product/dto/productdto_activevariantproduct.go

```diff
@@ -89,6 +89,17 @@ func (avp ActiveVariantProduct) Loyalty() ProductLoyalty {
 	}
 }
 
+func (avp ActiveVariantProduct) AvailableLoyalties() []ProductLoyalty {
```

Comment:
Why don't we just call `AvailableLoyalties` on the `ActiveVariant()`?

Reply:
done in the same way as single loyalty done, and available loyalties still gonna need packing into ```ProductLoyalty```